src/main/fc/tasks.c

```diff
@@ -378,7 +378,7 @@ cfTask_t cfTasks[TASK_COUNT] = {
     [TASK_MAIN] = DEFINE_TASK("SYSTEM", "UPDATE", NULL, taskMain, TASK_PERIOD_HZ(1000), TASK_PRIORITY_MEDIUM_HIGH),
     [TASK_SERIAL] = DEFINE_TASK("SERIAL", NULL, NULL, taskHandleSerial, TASK_PERIOD_HZ(100), TASK_PRIORITY_LOW), // 100 Hz should be enough to flush up to 115 bytes @ 115200 baud
     [TASK_BATTERY_ALERTS] = DEFINE_TASK("BATTERY_ALERTS", NULL, NULL, taskBatteryAlerts, TASK_PERIOD_HZ(5), TASK_PRIORITY_MEDIUM),
-    [TASK_BATTERY_VOLTAGE] = DEFINE_TASK("BATTERY_VOLTAGE", NULL, NULL, batteryUpdateVoltage, TASK_PERIOD_HZ(50), TASK_PRIORITY_MEDIUM),
+    [TASK_BATTERY_VOLTAGE] = DEFINE_TASK("BATTERY_VOLTAGE", NULL, NULL, batteryUpdateVoltage, TASK_PERIOD_HZ(200), TASK_PRIORITY_MEDIUM),
```

Comment:
I think so.  

There is a lot of jitter in the battery voltage measure.   I think the noise component is fairly high frequency and random.

The idea of this PR is to respond very quickly to the battery sag, so that there is no perceptible lag in the compensation.  The ideal total delay would be 15ms or less.  

We would like to have the least sampling delay as possible.  50hz sampling incurs a 20ms sampling delay.  That is far too much if our target delay is less than 15ms.  Even 200hz has a 5hz sampling delay, which is appreciable in this context.

If the lowpass period is set to 1 in the CLI, its period is 100ms, or a frequency 10hz.  This means a time constant is about 16ms. I have flown this and it is very good.  A lowpass at 10hz won't be an effective filter over data sampled at only 50hz. It will let a lot of jitter through, and it does.  A 10 hz lowpass is a barely acceptable filter for data at 200hz.

The default lpf period is 2, or 200ms, meaning a 5hz lowpass, which still lets a little jitter through with 200hz sampling, and has a time constant of around 30ms, which is a bit slow, but is usable.

It would be better still if we sampled at 400 or 500hz, then we could get delays closer to or inside our target delay.  200hz is a compromise; 50hz is not good enough.